package.json

```diff
@@ -49,19 +46,19 @@
     "@ember/optional-features": "^2.0.0",
     "@ember/test-helpers": "^2.4.2",
     "@ember/test-waiters": "^4.1.0",
-    "@embroider/test-setup": "^0.43.5",
+    "@embroider/test-setup": "^4.0.0",
     "@glimmer/component": "^1.0.4",
     "@glimmer/tracking": "^1.0.4",
     "active-model-adapter": "^3.0.1",
     "babel-eslint": "^10.1.0",
     "broccoli-asset-rev": "^3.0.0",
     "ember-cli": "~3.28.0",
     "ember-cli-dependency-checker": "^3.2.0",
-    "ember-cli-deprecation-workflow": "^1.0.1",
+    "ember-cli-deprecation-workflow": "^3.3.0",
     "ember-cli-inject-live-reload": "^2.1.0",
     "ember-cli-sri": "^2.1.1",
     "ember-cli-terser": "^4.0.2",
-    "ember-data": "~3.24",
+    "ember-data": "~3.28.13",
```

Comment:
Needed to avoid use of Ember global - later ember versions remove the ember global